printing/printer.ml

```diff
@@ -417,7 +417,8 @@ and bld_sign_env_id env sigma ctxt pps is_start =
 (* compact printing an env (variables and de Bruijn). Separator: three
    spaces between simple hyps, and newline otherwise *)
 let pr_context_limit_compact ?n env sigma =
-  let ctxt = Termops.compact_named_context (named_context env) in
+  let ctxt = Named.map (Evarutil.nf_evars_universes sigma) (named_context env) in
```

Comment:
Rather than normalizing here it should be the duty of compact_named_context to take an evarmap and to check using evar-aware equality. AFAICT we always have an evarmap at hand when calling this function.